ci-operator/config/openshift/knative-eventing-contrib/openshift-knative-eventing-contrib-release-v0.9.0__variant.yaml

```diff
@@ -85,22 +77,6 @@ images:
       - destination_dir: .
         source_path: /go/bin/camel-source-controller
   to: knative-eventing-sources-camel-source-controller
-- dockerfile_path: openshift/ci-operator/knative-images/websocketsource/Dockerfile
```

Comment:
I see these images still here:

https://github.com/knative/eventing-contrib/tree/v0.9.0/cmd

and in ours:

https://github.com/openshift/knative-eventing-contrib/tree/release-v0.9.0/cmd